pkgs/development/python-modules/rumps/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  pyobjc-framework-Cocoa,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "rumps";
+  version = "unstable-2025-02-02";
```

Comment:
```suggestion
  version = "0-unstable-2025-02-02";
```

Reply:
`version = "unstable-YYYY-MM-DD";` is the norm. See https://github.com/search?q=repo%3ANixOS%2Fnixpkgs%20version%20%3D%20%22unstable&type=code

Reply:
No, it is the old norm. Please read the contributing guide.

Reply:
there's no mention of this in https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md